finesse/gui/interferometer_monitor.py

```diff
@@ -127,10 +88,22 @@ def _display_props(self) -> None:
             lineedit = self._get_prop_lineedit(prop)
             lineedit.setText(prop.val_str())
 
-    def poll_server(self) -> None:
+    def get_data_table(self, data: list[EM27Property]):
+        """Receive the data table from the server."""
+        self._data_table = data
+
```

Comment:
Maybe call this `set_data_table`. In any case, if this is all it is doing, why not making `data_table` public (ie. remove the underscore) and just assigned `data` to it in the calling function?